src/http/api/resources/resolve.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
   async handler (request, h) {
     const { ipfs } = request.server.app
     const name = request.query.arg
-    const recursive = request.query.r || request.query.recursive || false
+    const recursive = request.query.r || request.query.recursive || true
```

Comment:
If `request.query.recursive` is explicitly set to `false`, will this code result in `recursive` being set to `true`?

Reply:
hummm good catch